### ISSUE TITLE
jansi 1.12 fails on Alpine Linux

### ISSUE THREAD

**Issue by rhuss** [fusesource/jansi#58]
```
Caused by: java.lang.UnsatisfiedLinkError: Could not load library. Reasons: [no jansi64-1.12 in java.library.path, no jansi-1.12 in java.library.path, no jansi in java.library.path, /tmp/libjansi-64-1-7763420971930555451.12: Error relocating /tmp/libjansi-64-1-7763420971930555451.12: ttyslot: symbol not found]
    at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:182)
    at org.fusesource.hawtjni.runtime.Library.load(Library.java:140)
    at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:42)
    at org.fusesource.jansi.AnsiConsole.wrapOutputStream(AnsiConsole.java:48)
```

Seem that the symbol "ttyslot" that is referred in the shared lib which is missing on Alpine Linux which does not use libc but musl.

jansi 1.11 works without problems.

If would be nice if jansi would be supported on Alpine as well since this distro is often used in a base Docker image.

**Comment by michael-o** [comment]
You need to fix Jansi native for that.

**Comment by bdw429s** [comment]
So I see this ticket was closed with a flurry of references to other tickets, but I don't see what the resolution was.  Is this fixed, and if so, in what version?  Thx.

**Comment by gnodet** [comment]
The ttyslot method has been removed a few weeks ago.
I haven't released jansi 1.16 yet, but I'm working on it.

**Comment by bdw429s** [comment]
Thank you @gnodet, looking forward to the release. Cc/ @jclausen
